src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -11,9 +11,26 @@ Released 2023-Sep-05
 Released 2023-Aug-21
 
 * **Breaking change**: Excluded attributes corresponding to `LogRecord.EventId`,
-`LogRecord.CategoryName` and `LogRecord.Exception` from the exported data. This
-is done as the semantic conventions for these attributes are not yet stable.
-([#4781](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4781))
+`LogRecord.CategoryName` and `LogRecord.Exception` from the exported data. See
```

Comment:
nit:

`LogRecord.Exception`, `LogRecord.EventId`, and
`LogRecord.CategoryName` - this order to be consitent.